src/plays/typing-speed-test/components/Stats.js

```diff
@@ -0,0 +1,25 @@
+const Stats = ({ stats: { wpm, cpm, accuracy } }) => {
+  let statsCard =

```

Comment:
are you changing the value of `statsCard` somewhere? If not, let's make it a `const`.

Reply:
Done